dependent/acta-veterinaria-scandinavica.csl

```diff
@@ -1,18 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
-  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/bmc -->
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
     <title>Acta Veterinaria Scandinavica</title>
-    <title-short>Acta Vet Scand</title-short>
     <id>http://www.zotero.org/styles/acta-veterinaria-scandinavica</id>
     <link href="http://www.zotero.org/styles/acta-veterinaria-scandinavica" rel="self"/>
-    <link href="http://www.zotero.org/styles/biomed-central" rel="independent-parent"/>
-    <link href="http://www.actavetscand.com" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/springer-basic-brackets" rel="independent-parent"/>
```

Comment:
Seems wrong. http://www.actavetscand.com/authors/instructions/research#formatting-references links to "SpringerVancouverNumber.ens", and examples look more like springer-vancouver-brackets.csl. (the original "biomed-central.csl" is wrong too)